app/src/gql/index.js

```diff
@@ -136,6 +136,7 @@ export const NetworksAll = gql`
         viewDenom
         chainToViewConversionFactor
       }
+      rpc_url
```

Comment:
hmm doesn't work as the rpc usually is whitelisted to the BE only. let's create a dictionary in the FE for those?

Reply:
Ah no sorry, I didn't update, it is working now

Reply:
no I mean we need to use the public polkadot endpoints in the FE

Reply:
See my comment below: https://github.com/luniehq/lunie/pull/4266#discussion_r444560139